libs/backend/src/cast_vote_records/export.ts

```diff
@@ -162,13 +158,6 @@ export interface RejectedSheet {
  */
 export type Sheet = AcceptedSheet | RejectedSheet;
 
-/**
- * An error encountered while exporting cast vote records to a USB drive
- */
-export type ExportCastVoteRecordsToUsbDriveError =
-  | { type: 'invalid-sheet-found'; message: string }
-  | ExportDataError;
```

Comment:
Likewise had to move this type to `libs/types`. The version in `libs/types` does not include a `message` field